cmd/contrib/main.go

```diff
@@ -180,15 +180,22 @@ func newPlugin() error {
 		}
 	}
 
-	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting
-	// from the last word. For example:
+	// As a placeholder, assume the field name is the short version (max 7 chars) of the credential name, starting from the last word.
+	//
+	// For example:
 	// "Personal Access Token" => "Token"
 	// "Secret Key" => "Key"
 	// "API Key" => "API Key"
+	//
+	// When the last word of the credential name is greater than seven characters, the last word is used as the field name
```

Comment:
super nit: we could move this comment line above the examples so we could also introduce an example `"Credentials" => "Credentials"`